### ISSUE TITLE
linting should prompt for an action on upcoming unreplied-to events ("needsAction")

### ISSUE THREAD

**Issue by flavorjones** [flavorjones/calendar-assistant#9]

**Comment by mikfreedman** [comment]
is this the first linting story @flavorjones ?

**Comment by flavorjones** [comment]
yes, I tried to put the "to do" backlog into a nice order. feel free to grab something else if you want but I think this is good to do first because:

* linting is contained to examining the event (and not something more complicated like "this meeting conflicts with another meeting" or "hours of meetings in this day")
* the update action is a chance to flesh out some of the behaviors around responding to a meeting, including
  * yes/no/maybe UX
  * send notifications? (I think we should default to yes for now)
  * testing how the EventRepository feels for a simple update like this

**Comment by flavorjones** [comment]
Here's what I had in my head for UX:

<pre>
<b>$</b> calendar-assistant lint today
<i>me@example.com</i>

<b>An Event is awaiting your response:</b>

  2018-10-01  10:30 - 10:55<b> | Facilitate out-of-the-box deliverables</b><i> (1:1, recurring)</i>
  Attendees:
  • foo@example.com (accepted)
  • bar@example.com (maybe)
  • baz@example.com (declined)
  • quux@example.com (awaiting)

Response? (<b>S</b>kip / <b>a</b>ccept / <b>m</b>aybe / <b>d</b>ecline) &gt; A

  Accepting <b>Facilitate out-of-the-box deliverables</b><i> (1:1, recurring)</i> ...


<b>An Event is awaiting your response:</b>

  2018-10-01  11:00 - 11:30<b> | Syndicate plug-and-play systems</b><i> (recurring)</i>
  Attendees:
  • foo@example.com (accepted)
  • bar@example.com (maybe)
  • baz@example.com (declined)
  • quux@example.com (awaiting)

Response? (<b>S</b>kip / <b>a</b>ccept / <b>m</b>aybe / <b>d</b>ecline) &gt; <i>hit return for default</i>

  Skipping Syndicate plug-and-play systems</b><i> (recurring)</i> for now.


<b>An Event is awaiting your response:</b>

  2018-10-01  11:30 - 12:00<b> | Facilitate bricks-and-clicks methodologies</b><i> (1:1, recurring)</i>
  • foo@example.com (accepted)
  • bar@example.com (maybe)
  • baz@example.com (declined)
  • quux@example.com (awaiting)

Response? (<b>S</b>kip / <b>a</b>ccept / <b>m</b>aybe / <b>d</b>ecline) &gt; D

  Declining <b>Facilitate out-of-the-box deliverables</b><i> (1:1, recurring)</i> ...


<b>No other events require your attention.</b> Have a great day!
</pre>

**Comment by flavorjones** [comment]
So yeah, maybe we should break this down into smaller stories, similar to how I've done it in #18 having multiple stories link back to it?

**Comment by mikfreedman** [comment]
Ar the very least I think there’s a read only story that can go first
